controllers/slicegateway/slicegateway.go

```diff
@@ -716,12 +749,15 @@ func (r *SliceGwReconciler) createEndpointForGatewayServer(slicegateway *kubesli
 			{
 				Addresses: []corev1.EndpointAddress{
 					{
-						IP: slicegateway.Status.Config.SliceGatewayRemoteNodeIP,
+						IP: slicegateway.Status.Config.SliceGatewayRemoteNodeIPs[0],
```

Comment:
why is only 1st node IP appended in endpoint address?

Reply:
If you check the next few lines, I am creating another EndpointAddress object and appending it to the Subsets only if there are two RemoteNodeIPs Present/available.